debian/sid/debian/patches/add-docs-rst-readme

```diff
@@ -0,0 +1,49 @@
+Description: Add a top-level README for RST documentation
+ This package installes only RST documentation and not yet HTML
+ documentation. For that reason, the existing top-level README needed to
+ be modified. Additionally note that the examples/ directory is available
+ there.
+Author: Ben Albrecht <balbrecht@cray.com>
+Forwarded: not-needed
+diff --git a/doc/debian/README.rst b/doc/debian/README.rst
```

Comment:
I'm a bit confused about the pathing here... Should this replace `doc/README.rst`?